src/components/InfoPanel/InfoPanel.js

```diff
@@ -23,11 +36,25 @@ const InfoPanel = () => {
     ? extractCloudName(modelStatusData.model.cloudTag)
     : "";
 
+  const { width, height } = expandedTopologyDimensions();
+
   return (
     <div className="info-panel">
-      <div className="info-panel__pictogram">
-        <Topology width={300} height={300} modelData={modelStatusData} />
-      </div>
+      {showExpandedTopology ? (
+        <Modal close={() => setShowExpandedTopology(false)} title={false}>
```

Comment:
It would be nice to pass the bundle name as the title.

Reply:
This isn't a bundle, it's the model which may or may not have one or many bundles in it :)

Reply:
Sorry, I meant model name.